doc/en/components/grids/_shared/collapsible-column-groups.md

```diff
@@ -126,30 +124,34 @@ Default collapse indicator for the `{ComponentName}` is the following:
 
 Also, if you need to change the default expand/collapse indicator, we provide templating options in order to achieve this.
 
-### Using Property
-
-You can define custom expand/collapse template and provide it to each of the collapsible column groups using `CollapsibleIndicatorTemplate` property. Check the markup below:
+```razor
+ <IgbColumnGroup @ref="infoColumn" Header="Customer Information" Collapsible="true">
+    <IgbColumn Field="CustomerName" Header="Full name" VisibleWhenCollapsed="true"></IgbColumn>
+    <IgbColumn Field="CustomerID" Header="Customer ID" VisibleWhenCollapsed="false"></IgbColumn>
+    <IgbColumn Field="FirstName" Header="First Name" VisibleWhenCollapsed="false"></IgbColumn>
+    <IgbColumn Field="LastName" Header="Last Name" VisibleWhenCollapsed="false"></IgbColumn>
+    <IgbColumnGroup @ref="addressColumn" Header="Customer Address">
+        <IgbColumn Field="Country" Header="Country" Sortable="true"></IgbColumn>
+        <IgbColumn Field="City" Header="City" Sortable="true"></IgbColumn>
+    </IgbColumnGroup>
+ </IgbColumnGroup>
 
-```html
-<ng-template #indTemplate let-column="column">
-    <igx-icon [attr.draggable]="false" >{{column.expanded ? 'remove' : 'add'}} </igx-icon>
-</ng-template>
-<igx-column-group header="Customer Information" [collapsible]="true" [collapsibleIndicatorTemplate]="indTemplate">
-    <igx-column field="CustomerName" header="Fullname" [dataType]="'string'" [visibleWhenCollapsed]="true"></igx-column>
-    <igx-column field="CustomerID" header="Customer ID" [dataType]="'string'" [visibleWhenCollapsed]="false"></igx-column>
-    <igx-column-group header="Customer Address" [collapsible]="true" [collapsibleIndicatorTemplate]="indTemplate">
-        <igx-column field="Country" header="Country" [dataType]="'string'" [sortable]="true" [visibleWhenCollapsed]="true"></igx-column>
-        <igx-column field="City" header="City" [dataType]="'string'" [sortable]="true" [visibleWhenCollapsed]="false"></igx-column>
-    </igx-column-group>
-</igx-column-group>
-```
+@code {
+    private IgbColumnGroup infoColumn;
+    private IgbColumnGroup addressColumn;
 
-```razor
-    public RenderFragment<IgbColumnTemplateContext> Template = (context) =>
+    public RenderFragment<IgbColumnTemplateContext> ColumnIndicatorTemplate = (context) =>
```

Comment:
This does work to some extend with the blazor template, but it gets evaluated just once and on expand/collapse does not really change the template based on the column state:
![template](https://github.com/IgniteUI/igniteui-xplat-docs/assets/10397980/cd227553-93f0-478f-9e7a-d725008912bc)
This is probably a blazor issue as it works ok in web components. Not sure if we should keep the snippet or skip it for now as it does not fully work yet.